bindings_ffi/Makefile

```diff
@@ -1,9 +1,11 @@
+PROJECT_NAME = xmtpv3
+
 # Simulator config
 ARCHS_IOS = x86_64-apple-ios aarch64-apple-ios-sim
 ARCHS_MAC = x86_64-apple-darwin aarch64-apple-darwin
```

Comment:
IIUC this is because of the lipo vs xcframework stuff. If you just try to combine all the binaries into the xcframework, it will complain that some libraries look the same. It seems that what you need to do is to combine different cpu arch for the same platform using lipo, and then combine these combined/fat binaries into an xcframework. I have a link that discusses that in one of the follow up comments here.

Reply:
https://developer.apple.com/forums/thread/666335